src/constants.rs

```diff
@@ -15,6 +15,12 @@ lazy_static! {
     pub(crate) static ref PROJECT_NAME: &'static str = env!("CARGO_PKG_NAME");
     pub(crate) static ref PROJECT_VERSION: &'static str = env!("CARGO_PKG_VERSION");
     pub(crate) static ref VERSION_CODENAME: &'static str = include_str!("../CODENAME");
+    pub(crate) static ref CONFIG_PATH: String = {
+        match std::env::var("CARNIVAL_CONFIG_PATH") {
+            Ok(p) => String::from(p),
+            Err(_e) => "".to_string()
```

Comment:
Again, I think we should enforce knowing the actual config path

Reply:
Here it's OK. since it being empty or non-existent is used for the default path.

Reply:
And what would the default path be? Can you share the relevant docs?

Reply:
I would prefer if this was Option then